src/main/resources/db/migration/V3__Create_job_roles_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE job_roles (
+    jobRoleId INT PRIMARY KEY,
+    roleName VARCHAR(255) NOT NULL,
```

Comment:
Please count chars in "Principal workday integration solution architect for smart product" -> I don't know longer role :D And limit to sth around the count